src/components/standalone/dns_dhcp/DynamicLeases.vue

```diff
@@ -5,18 +5,42 @@
 
 <script setup lang="ts">
 import { useI18n } from 'vue-i18n'
-import FilterableListItemLayout from '../FilterableListItemLayout.vue'
 import { ubusCall } from '@/lib/standalone/ubus'
 import LeasesTable from './LeasesTable.vue'
-import { ref } from 'vue'
+import { ref, computed, onMounted } from 'vue'
 import { useUciPendingChangesStore } from '@/stores/standalone/uciPendingChanges'
 import { getStandaloneRoutePrefix } from '@/lib/router'
 import { useRouter } from 'vue-router'
 import CreateOrEditStaticLeaseDrawer from './CreateOrEditStaticLeaseDrawer.vue'
-
+import {
+  NeButton,
+  NeEmptyState,
+  NeInlineNotification,
+  NeSkeleton,
+  NeTextInput,
+  getAxiosErrorMessage,
+  type FilterOption,
+  NeDropdownFilter
+} from '@nethesis/vue-components'
+import { faCircleInfo } from '@fortawesome/free-solid-svg-icons'
 const { t } = useI18n()
-const uciChangesStore = useUciPendingChangesStore()
 const router = useRouter()
+// Define props
+const props = defineProps<{
+  fetchErrorNotificationTitle?: string
+}>()
+const selectedLease = ref<DynamicLease>()
+const showCreateStaticLeaseDrawer = ref(false)
+const uciChangesStore = useUciPendingChangesStore()
+const filter = ref('')
```

Comment:
I suggest to rename this to `textFilter` for clarity, since we have multiple filters now